### PR TITLE
use schema defined by cr8 tool for benchmark api

### PR DESCRIPTION
**Pull request by chaudum**
@bogensberger can you take a look please!

### CODE REVIEW

File: api/README.rst [hunk 1]
```diff
@@ -35,44 +35,66 @@ REST Endpoint::
...
 
 Table Schema
 ============
 
-::
+The table schema is defined by the `cr8`_ tool.
+
+.. code-block:: sql
 
   CREATE TABLE IF NOT EXISTS "benchmark"."history" (
```
Comment (bogensberger) [inline api/README.rst]: i would remove the schema here and just link to `cr8`

File: api/config.toml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+debug = true
+crate_hosts = [
+#  "st1.p.fir.io:4200",
+#  "st2.p.fir.io:4200",
+#  "st3.p.fir.io:4200",
+#  "st4.p.fir.io:4200",
```
Comment (bogensberger) [inline api/config.toml]: remove uncomment hosts

File: api/schema.sql [hunk 3]
```diff
@@ -0,0 +1,36 @@
+CREATE TABLE IF NOT EXISTS "benchmark"."history" (
```
Comment (bogensberger) [inline api/schema.sql]: isn't it possible to use it from cr8? So we don't have to update it here if a new cr8 version brings a new schema

File: api/README.rst (hunk 1 above)
Comment (chaudum) [inline api/README.rst]: there are 2 minor changes to the schema from `cr8`:
* table name
* version_info['timestamp']

Comment (bogensberger) [inline api/README.rst]: ok, then ignore my comments about schema

**Comment by bogensberger** [comment]
LGTM

**Pull request merged**